packages/wallets/src/features/accounts/modules/DocumentService/components/Onfido/Onfido.tsx

```diff
@@ -39,7 +41,13 @@ const Onfido: React.FC<TOnfidoProps> = ({ onCompletion }) => {
     if (isLoading) return <Loader />;
 
     return (
-        <ModalStepWrapper title={localize('Add a real MT5 account')}>
+        <ModalStepWrapper disableAnimation={!!onClickBack} title={localize('Add a real MT5 account')}>
+            {onClickBack && (
+                <button className='wallets-onfido__back-button' onClick={onClickBack}>
+                    <LegacyArrowLeft2pxIcon iconSize='xs' />
+                    <Text weight='bold'>Back</Text>
```

Comment:
Should this be localized @aum-deriv?
```suggestion
                    <Text weight='bold'>Back</Text>
```